src/colmap/sfm/incremental_mapper.h

```diff
@@ -242,7 +242,16 @@ class IncrementalMapper {
   size_t FilterImages(const Options& options);
   size_t FilterPoints(const Options& options);
 
-  const Reconstruction& GetReconstruction() const;
+  // Getter functions
+  const std::shared_ptr<Reconstruction>& GetReconstruction() const {
```

Comment:
Same comments as in the previous PR: why adding a `Get*` prefix? The codebase doesn't follow this convention anywhere so, please, to remain consistent and minimal, let's not add it.

Reply:
Thanks. This is updated.